src/components/Time/Time.js

```diff
@@ -1 +1,5 @@
-export const getFinishTime = times => times[times.length - 1].time;
+const validateTime = time => time ? time.time : 0;
+
+export const getFinishTime = times => validateTime(times.find(time => time.type === 'Finish'));
+
+export const getSplash = times => distance => validateTime(times.find(time => time.distance === distance && time.type === 'Clocking location'));
```

Comment:
Eigenlijk van deze string een const gaan maken denk ik.

Reply:
Maar dan zoiets?
https://stackoverflow.com/questions/40896261/apply-global-variable-to-vuejs

Reply:
Of gewoon een bestand als locationTypes.js met daarin:

```export const finish = 'Finish';
export const splash = 'Clocking location'``` ect..